applications/wg-easy/Taskfile.yaml

```diff
@@ -172,6 +172,36 @@ tasks:
     deps:
       - cluster-create
 
+  helm-preflight-dryrun:
+    desc: Verify the preflight specs that will be run
+    silent: false
+    cmds: 
+      - |
+        TEMP_FILE=$(mktemp)
+        
+        # Find all charts and append their templates to the temp file
+        for chart_dir in $(find charts/ -maxdepth 1 -mindepth 1 -type d); do
+          helm template "$chart_dir" >> "$TEMP_FILE"
+          echo "---" >> "$TEMP_FILE" # Add separator between charts
+        done
+        
+        # Run preflight once on the combined templates
+        echo "Running preflight checks on all templates"
+        cat "$TEMP_FILE" | kubectl preflight - --dry-run
+        
+        # Clean up
+        rm "$TEMP_FILE"
+
+  helm-preflight:
+    desc: Run preflight checks on Helm charts using preflight CLI
+    silent: false
+    cmds: 
+      - defer: rm -f preflightbundle-*.tar.gz
```

Comment:
The fact that Taskfile can 'defer' is pretty nice I didn't know that before your changes.

However, isn't removing the preflight counter productive? If a preflight fails the user may want to look at the failure or provide it to support for help. At the very least shouldn't there be a way to not remove it so that it's available?

I think my personal preference would be to either:
1. Remove preflightbundle-*.tar.gz before we run the preflight so that only the most recent bundle is present.
2. Don't even try to remove them, just make sure it's in .gitignore and let people clean them up as they see fit.

Reply:
Gotcha, I opted in for 2 and updated .gitignore